app/assets/stylesheets/course/enrol_requests.css.scss

```diff
@@ -0,0 +1,44 @@
+$link-color: #0000ff;
+
+#enrol-request-form {
```

Comment:
group your selectors according to the controller/action paradigm that winstonyw/rails_utils@1752d6eac348681fb00720538e9b8cb784f935ab uses